doc/ref_model/chapters/chapter02.md

```diff
@@ -266,16 +266,44 @@ By trying to sort workloads into different categories based on the requirements
+- For a given profile, **flavours** represent small deviations from (or further qualification, such as sizing) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, i.e.: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour. Workloads requesting a given flavour **must** be instantiated on infrastructure with that same flavour. It is allowed to instantiate workloads on infrastructure with more flavours than what is requested, as long as the minimum requirements are satisfied.
+
+### 2.4.1 Infrastructure profiles (top-level)
+
+Based on the above analysis, the following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
 - **Basic**: for Workloads that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for Workloads that require predictable computing performance, high network throughput and low network latency.
+- **High Performance**: for Workloads that require predictable computing performance, high network throughput and low network latency.
 
 <p align="center"><img src="../figures/ch02_infra_profiles.PNG" alt="infra_profiles" title="Infrastructure Profiles" width="100%"/></p>
 <p align="center"><b>Figure 2-1:</b> Infrastructure profiles proposed based on VNFs categorisation.</p>
 
-In **[Chapter 4](./chapter04.md)** these **B (Basic)** and **N (Network intensive)** infrastructure profiles will be defined in greater detail for use by workloads:
+In **[Chapter 4](./chapter04.md)** these **B (Basic) Performance** and **H (High) Performance** infrastructure profiles will be defined in greater detail for use by workloads.
+
+Profiles partition the infrastructure: an infrastructure object (host/node) **must** have one and only one profile associated to it.
+
+### 2.4.2 Node flavours
+
+Node flavours are meant to be used as labels for infrastructure, identifying the nodes that implement special capabilities that go beyond the profile baseline. Certain flavours may be relevant only for some profiles.
+The following **node flavours** are proposed:
+
+- **High performance CPU**: for Workloads that require predictable computing performance (or higher clock speed).
+- **High performance storage**: for Workloads that require low storage latency and/or high storage IOPS.
+- **High memory**: for Workloads that require high amounts of RAM.
+- **GPU**: denotes the presence of a consumable GPU on the infrastructure
+- **High performance network (25G)**: denotes the presence of network links (to the DC network) of speed of 25 Gbps or greater.
```

Comment:
Is 25G high performance. Also should this and the 100G be "High speed network"

or
"Min 25G speed network" and "Min 100G speed network"